### ISSUE TITLE
Unable to delete files

### ISSUE THREAD

**Issue by asabya** [fairDataSociety/fairdrive-theapp#249]
![telegram-cloud-photo-size-5-6176745820641146577-y](https://user-images.githubusercontent.com/15252513/210736891-368d8762-c110-4008-8c5c-908485916b8d.jpg)

"File could not be deleted"

**Comment by tomicvladan** [comment]
@asabya Did you try to delete some other file? Or do all files fail?

**Comment by asabya** [comment]
At that point of time all files were failed. Tried now also...

- tried to delete one file.. after a while delete confirmation modal went away. but the file stayed.
- Then tried to delete the same file, got "File could not be deleted!"

**Comment by tomicvladan** [comment]
Which environment are you using? I've tried on `https://app.fairdrive.fairdatasociety.org/` and files delete normally.

**Comment by asabya** [comment]
I was using staging. Isn't that updated?

**Comment by tomicvladan** [comment]
Yes, it's updated today. I've just tried staging as well, and I was able to delete a file without problems. Maybe you were testing with the old app, or your account got corrupted somehow.

**Comment by asabya** [comment]
For me the issue still persists on
https://app.fairdrive.fairdatasociety.org/drive aswell
